### PR TITLE
Test handle_frontend functions and implement backend functionality for subscribing / unsubscribing from drop, and sharing drop info

### PR DESCRIPTION
**Pull request by JLaRue96**
The title includes the tentative goals of this week for me. There will be stuff added to this throughout the night tonight. Please review this tomorrow morning.

### CODE REVIEW

**Comment by JLaRue96** [comment]
Just wanted to get a PR set up. Nothing added yet.

**Comment by JLaRue96** [comment]
@matthewbentley Yo I'm getting a weird travis error where 'syncr_backend.network' isn't being recognized as a module. Do you know what might be causing this?

**Comment by JLaRue96** [comment]
Gonna do a wee bit more work today to get backend working for subscribing to a drop. This'll be done by the early afternoon at the latest.

File: syncr_backend/network/handle_frontend.py [hunk 1]
```diff
@@ -495,24 +496,32 @@ def handle_frontend_request(
...
-        # TODO: backend logic to subscribe to drop.
-        # TODO: Test if given drop_name is valid.
-        response = {
-            'status': 'ok',
-            'result': 'success',
-            'message': 'subscribed to drop ' + request['drop_name'],
-        }
+
+        try:
+            await sync_drop(request['drop_id'], request['file_path'])
```
Comment (matthewbentley) [inline syncr_backend/network/handle_frontend.py]: This actually does the whole sync, not just queues it.  So this page will not return until the sync finishes.  The better option would be to do `asyncio.ensure_future(sync_drop(...))` for now.

**Pull request merged**